CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [0.9.2](https://github.com/haskell-nix/hnix/compare/0.9.1...0.9.2) (yyyy-mm-dd)
+
+* Additional:
+  * Dependencies:
```

Comment:
@Anton-Latukha, what does "Additional" mean here? Is this the heading for a section of _additions_ as opposed to _breaking changes_?

In this case, I think it is easy to misinterpret this wording as "additional dependencies".

How about something like this instead!?


```suggestion
* Dependency updates:
```

Reply:
Yep, `Additional`, since:
1. Only particular changes go into API breakages, and so in `Breaking` changes.
2. In "Minor" changes people include changes that are not breaking, it is one size fits all section, people are expected to include not only particular different things into it. `Minor` bears hidden knowledge of what `minor` means in Haskell, but Haskells `minor` does not include project-related changes. `Different` - includes the breadth, but does not include specifics of application regarding the API, word does not underline that changes may be skipped over by the reader. So that is why `Additional`, word that means one-size-fits-all (`different`) & `minor` at the same time.

Reply:
The changelog structure should be consistent, at least to respect the eye and the mind of the reader. So there always should be the same headlines that give the same most important message to the reader, an explicit `Breaking` section that includes the most important to reader information, and `Additional` that includes all other info project community conveys.

Reply:
@Anton-Latukha The changelog structure that you seem to be describing sounds rather unconventional to me, so I think people might have difficulties understanding it.

In particular, using "additional" as a short-hand for "different" (I think this section is often labeled "other changes" in other changelogs) and "minor [changes]", seems extremely confusing to me.

Regarding consistency, I agree that it is important, but not more important than clarity.

Reply:
Yep, "other changes".